src/services/timerFunction.ts

```diff
@@ -0,0 +1,214 @@
+import moment, { Moment } from "moment";
+import {
+  addFinishedExecution,
+  addOngoingExecution,
+  getOngoingExecutions,
+  setOngoingExecution,
+  removeOngoingExecution,
+} from "./localStorage";
+import { ExecutionStatus, OngoingExecution, FinishedExecution } from "./types";
+
+/**
+ * Coordena o a manipulação de execuções.
+ *
+ * O objetivo é centralizar as funções relacionadas à execuções, de forma a
+ * não poluir as Screens que em que serão utilizadas e que mudanças necessárias
+ * sejam automaticamente propagadas a todas as telas que necessitem trabalhar com
+ * execuções.
+ *
+ */
+
+/** Função responsável por criar uma nova execução */
+export const createExecution = async (cardInfo: any) => {
+  let startTime: Moment = moment();
+  let strStartTime: string = startTime.format("YYYY-MM-DDTHH:mm:ss[Z]ZZ");
+  console.log(
+    "timerFunction.createExecution: Criando execução - Horário atual: ",
+    strStartTime
+  );
```

Comment:
Quando a execução é criada, ela não inicia automático, é necessário selecionar "INICIAR" pra ter um `startTime`. Então eu acho que nesse momento ele deve estar só zerado, certo?

```suggestion
  console.log(
    "timerFunction.createExecution: Criando execução"
  );
```

Reply:
Eu estava analisando o código e não me pareceu ser um problema, então vou manter como está

Reply:
o método initializeExecution garante a integridade do startTime e outras variáveis do objeto